resources/ts/types/Integration.ts

```diff
@@ -22,7 +22,7 @@ export type Integration = {
   name: string;
   description: string;
   subscriptionId: string;
-  coupon: Coupon;
+  coupon?: Coupon | null;
```

Comment:
I feel the first `?` should not be here since the property will always be there but might be null whereas here it can be both missing and null